lbrynet/conf.py

```diff
@@ -58,29 +65,72 @@ def _win_path_to_bytes(path):
             pass
     return path
 
-if sys.platform.startswith('darwin'):
-    platform = DARWIN
-    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
-    default_data_dir = user_data_dir('LBRY')
-    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
-elif sys.platform.startswith('win'):
-    platform = WINDOWS
-    from lbrynet.winhelpers.knownpaths import get_path, FOLDERID, UserHandle
 
-    default_download_directory = get_path(FOLDERID.Downloads, UserHandle.current)
-    default_data_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbrynet')
-    default_lbryum_dir = os.path.join(
-        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbryum')
+def _get_old_directories(platform):
+    dirs = {}
+    if platform == WINDOWS:
+        dirs['data'] = user_data_dir('lbrynet', roaming=True)
+        dirs['lbryum'] = user_data_dir('lbryum', roaming=True)
+        dirs['download'] = get_path(FOLDERID.Downloads, UserHandle.current)
+    elif platform == DARWIN:
+        dirs['data'] = user_data_dir('LBRY')
+        dirs['lbryum'] = os.path.expanduser('~/.lbryum')
+        dirs['download'] = os.path.expanduser('~/Downloads')
+    elif platform == LINUX:
+        dirs['data'] = os.path.expanduser('~/.lbrynet')
+        dirs['lbryum'] = os.path.expanduser('~/.lbryum')
+        dirs['download'] = os.path.expanduser('~/Downloads')
+    return dirs
+
```

Comment:
add

else:
   raise ValueError('unknown platform value') 

incase platform is set to some unknown value